Frontend/ui-common/src/Engineering/Helpers/Commons.js

```diff
@@ -493,6 +494,7 @@ function formatDates(date, format, language) {
     },
     'MMM': () => {
       const month = date.toLocaleDateString(language, { month: 'short' });
+      const month2 = date.toLocaleDateString(language, { month: 'long' });
```

Comment:
@prakharritik this variable is not being used anywhere. Is it required?

Reply:
Fixed